### PR TITLE
Improve geocoding flow

### PR DESCRIPTION
**Pull request by rjimenezda**
This PR improves the onboarding for users regarding layers without geographical information

The result:
disabled

Related ticket: #13046

### CODE REVIEW

**Comment by piensaenpixel** [comment]
After talking to Roman, we think  it makes sense go to the style tab when the analysis finishes.
Thanks!

**Comment by rjimenezda** [comment]
Finishes correctly, right?

File: lib/assets/core/test/spec/cartodb3/editor/layers/layer-content-view/analyses/analyses-view.spec.js [hunk 1]
```diff
@@ -230,14 +215,4 @@ describe('editor/layers/layer-content-views/analyses/analyses-view', function ()
       expect(this.view._isLayerHidden()).toBe(true);
     });
   });
-
-  describe('._onGeoreferenceClicked', function () {
-    it('should add Georeference Analysis', function () {
```
Comment (rubenmoya) [inline lib/assets/core/test/spec/cartodb3/editor/layers/layer-content-view/analyses/analyses-view.spec.js]: Aren't we testing this anymore?

Comment (rjimenezda) [reply lib/assets/core/test/spec/cartodb3/editor/layers/layer-content-view/analyses/analyses-view.spec.js]: 🤔 Let me check if I accidentally that particular test, I think it was for the infobox but now we have a different scenario

Comment (rjimenezda) [reply lib/assets/core/test/spec/cartodb3/editor/layers/layer-content-view/analyses/analyses-view.spec.js]: So, yeah, that was the old infobox that would show below. It's gone now, and the `click .js-georeference` is handled on another level 👍

**Comment by matallo** [comment]
merged ui states from master, and modified a bit the css when solving conflicts with @rubenmoya latest changes in the style guide

still in acceptance

**Comment by matallo** [comment]
restest this, please

**Comment by rjimenezda** [comment]
Merging master to fix the backend specs

**Comment by piensaenpixel** [comment]
vamos!

**Pull request merged**